pints/plot/_series.py

```diff
@@ -92,6 +92,8 @@ def series(samples, problem, ref_parameters=None, thinning=None):
 
     # Guess appropriate alpha (0.05 worked for 1000 plots)
     alpha = max(0.05 * (1000 / (n_sample / thinning)), 0.5)
+    if alpha > 1:  # pragma: no cover
```

Comment:
You can avoid the pragma by doing e.g. `alpha = min(1, max(0.05...))`

Reply:
Thanks -- I've addressed this in another PR since I needed it to pass the tests.